robot.py

```diff
@@ -129,6 +129,12 @@ def teleopInit(self):
         self.pistons_activated = False
         self.forward = 0
 
+        #Vision sensor
+        self.visionSensor = VisionSensor('127.0.0.1', 8812)
+
+        #Sonar sensor
+        self.sonarSensor = SonarSensor('127.0.0.1', 8813)
```

Comment:
Same issue here, no loopback to be committed to master.

Perhaps we could having something near the top:

if we_are_doing_simulation:
    SONAR_IP = '127.0.0.1'
else:
   SONAR_IP = '10.10.76.9'

Then we can keep this part of the code nice and clean.
While you are at, give the ports nice ALL_CAPS names as well.

Even if we can't figure out how to do that simulation thing, we could have our own
function  doing_simulation() that return False (unless we hack it...)